apps/frontend/src/app/components/header/header.component.scss

```diff
@@ -225,7 +231,21 @@
     }
   }
 
-  &:hover,
+  &:hover {
+    > a,
+    > span {
+      background-color: #f7f7f7;
+    }
+
+    .submenu {
+      opacity: 1;
+      pointer-events: all;
+
+      @media screen and (max-width: 992px) {
+        display: none;
+      }
+    }
+  }
```

Comment:
This modification doesn't achieve anything. And when I said that something broke another thing, I only meant it for only the line I commented on. Please revert.